summerofcode/2023.md

```diff
@@ -61,6 +61,7 @@ If you are a project maintainer and consider mentoring during the GSoC 2023 cycl
     + [GitHub Actions for KubeArmor](#github-actions-for-kubearmor)
     + [Store KubeArmor policies in OCI registry](#store-kubearmor-policies-in-oci-registry)
   * [Kubebuilder](#kubebuilder)
+    + [Leverage Phase 2 Plugins API to allow creation and usage of external plugins](#leverage-phase2-plugins-api)
```

Comment:
```suggestion
    + [Leverage Phase 2 Plugins API to allow creation and usage of external plugins](##leverage-phase-2-plugins-api-to-allow-creation-and-usage-of-external-plugins)
```

Reply:
Let's get fix the anchor and we're good to go

Reply:
Fixed it.